package.json

```diff
@@ -28,24 +28,24 @@
     "zone.js": "~0.8.26"
   },
```

Comment:
"dependencies": {
    "@angular/animations": "^6.1.0",
    "@angular/common": "^6.1.0",
    "@angular/compiler": "^6.1.0",
    "@angular/core": "^6.1.0",
    "@angular/forms": "^6.1.0",
    "@angular/http": "^6.1.0",
    "@angular/platform-browser": "^6.1.0",
    "@angular/platform-browser-dynamic": "^6.1.0"


You might want to check them again.... you can find them here https://www.npmjs.com/

Reply:
Please check the entire file. thanks :)